otx/cli/tools/train.py

```diff
@@ -90,16 +101,22 @@ def parse_args():
         required=False,
         help="Comma-separated paths to unlabeled file list",
     )
+
     parser.add_argument(
         "--load-weights",
         required=False,
         help="Load only weights from previously saved checkpoint",
     )
     parser.add_argument(
         "--save-model-to",
-        required="True",
+        required=False,
         help="Location where trained model will be stored.",
     )
+    parser.add_argument(
```

Comment:
Do we need to separate the `--save-model-to` and `--save-logs-to` ?
we might combine together like as `--output_path` then user don't need to add an additional argument.

Reply:
It looks like this will be a bigger task than I thought. (Several test code needs to be fixed.) After discussion with group, it seems appropriate to work on another PR. What do you think?

Reply:
Agreed